packages/feature-detector/src/find-globals.ts

```diff
@@ -0,0 +1,52 @@
+import { dirname, resolve } from 'path'
+import { Project, SourceFile, ts } from 'ts-morph'
+import { buildProject } from './utils/project'
+
+/**
+ * Find the list of globals used by source files in the provided project.
+ * Analyzed source files can be filtered by provided a list of glob patterns (default to all TypeScript and JavaScript files, excluding type definitions)
+ */
+export function findGlobals(
+  sourcePath: string,
+  project: Project = buildProject()
+): string[] {
+  const globals = new Set<string>()
+  const sourceFile = project.getSourceFileOrThrow(sourcePath)
+  addFileGlobals(sourceFile, globals)
+  for (const imported of sourceFile.getImportDeclarations()) {
+    console.log(imported.getModuleSpecifierSourceFileOrThrow())
+    const importedFile = project.getSourceFile(
```

Comment:
nit: Remove `console.log`

```suggestion
    const importedFile = project.getSourceFile(
```